node/node_test.go

```diff
@@ -418,7 +418,9 @@ func TestRegisterHandler_Successful(t *testing.T) {
 // Tests that the given handler will not be successfully mounted since no HTTP server
 // is enabled for RPC
 func TestRegisterHandler_Unsuccessful(t *testing.T) {
-	node, err := New(&DefaultConfig)
+	cfg := DefaultConfig
```

Comment:
I wonder why we even use the default config here. We usually use an empty config in tests, and it uses a temp data directory by default.

Reply:
The test is useless. It does not at all "test that the given handler will **not** be successfully mounted".
You can pass it the empty config, or use the same node as in `TestRegisterHandler_Unsuccessful`, which definitely registers a handler. 

The most that the test does, I guess, is "test that the registration does not `panic` when used with default vaues. 

IMO let's just drop it.